geometry_msgs/PoseWithCovarianceStamped pose
string identity